packages/clarity-js/src/data/metadata.ts

```diff
@@ -71,17 +81,27 @@ export function consent(): void {
 }
 
 export function clear(): void {
-  // Clear any stored information in the session so we can restart fresh the next time
-  if (supported(window, Constant.SessionStorage)) { sessionStorage.removeItem(Constant.StorageKey); }
+  // Clear any stored information in the session cookie so we can restart fresh the next time
+  setCookie(Constant.SessionKey, Constant.Empty, 0);
+}
+
+function tab(): string {
+  let id = shortid();
+  if (supported(window, Constant.SessionStorage)) {
+    let value = sessionStorage.getItem(Constant.TabKey);
+    id = value && value.indexOf(Constant.Separator) < 0 ? value : id;
+    sessionStorage.setItem(Constant.TabKey, id);
+  }
+  return id;
 }
 
 export function save(): void {
   let ts = Math.round(Date.now());
   let upgrade = config.lean ? BooleanFlag.False : BooleanFlag.True;
   let upload = typeof config.upload === Constant.String ? config.upload : Constant.Empty;
   if (upgrade && callback) { callback(data, !config.lean); }
-  if (config.track && supported(window, Constant.SessionStorage)) {
-    sessionStorage.setItem(Constant.StorageKey, [data.sessionId, ts, data.pageNum, upgrade, upload].join(Constant.Separator));
+  if (config.track && supported(document, Constant.Cookie)) {
+    setCookie(Constant.SessionKey, [data.sessionId, ts, data.pageNum, upgrade, upload].join(Constant.Separator), Setting.SessionExpire);
```

Comment:
this check is present in the setcookie function anyway

Reply:
Good point, will remove, thanks.

Reply:
Done